contract.js

```diff
@@ -41,154 +47,308 @@ web3.currentProvider.enable().then(() => {
+          approveTokens.disabled = false
+          transferTokensWithoutGas.disabled = false
+          approveTokensWithoutGas.disabled = false
+
+          transferTokens.onclick = (event) => {
             console.log(`event`, event)
             contract.transfer('0x2f318C334780961FB129D2a6c30D0763d9a5C970', '15000', {
-              from: web3.eth.accounts[0],
+              from: accounts[0],
               to: contract.address,
               data: '0xa9059cbb0000000000000000000000002f318c334780961fb129d2a6c30d0763d9a5c9700000000000000000000000000000000000000000000000000000000000003a98',
               gas: 60000,
               gasPrice: '20000000000',
-            }, function (result) {
+            }, (result) => {
               console.log('result', result)
             })
-          })
+          }
 
-          approveTokens.addEventListener('click', function () {
+          approveTokens.onclick = () => {
             contract.approve('0x9bc5baF874d2DA8D216aE9f137804184EE5AfEF4', '70000', {
-              from: web3.eth.accounts[0],
+              from: accounts[0],
               to: contract.address,
               data: '0x095ea7b30000000000000000000000009bc5baF874d2DA8D216aE9f137804184EE5AfEF40000000000000000000000000000000000000000000000000000000000000005',
               gas: 60000,
               gasPrice: '20000000000',
-            }, function (result) {
+            }, (result) => {
               console.log(result)
             })
-          })
+          }
 
-          transferTokensWithoutGas.addEventListener('click', function (event) {
+          transferTokensWithoutGas.onclick = (event) => {
             console.log(`event`, event)
             contract.transfer('0x2f318C334780961FB129D2a6c30D0763d9a5C970', '15000', {
-              from: web3.eth.accounts[0],
+              from: accounts[0],
               to: contract.address,
               data: '0xa9059cbb0000000000000000000000002f318c334780961fb129d2a6c30d0763d9a5c9700000000000000000000000000000000000000000000000000000000000003a98',
               gasPrice: '20000000000',
-            }, function (result) {
+            }, (result) => {
               console.log('result', result)
             })
-          })
+          }
 
-          approveTokensWithoutGas.addEventListener('click', function () {
+          approveTokensWithoutGas.onclick = () => {
             contract.approve('0x2f318C334780961FB129D2a6c30D0763d9a5C970', '70000', {
-              from: web3.eth.accounts[0],
+              from: accounts[0],
               to: contract.address,
               data: '0x095ea7b30000000000000000000000002f318C334780961FB129D2a6c30D0763d9a5C9700000000000000000000000000000000000000000000000000000000000000005',
               gasPrice: '20000000000',
-            }, function (result) {
+            }, (result) => {
               console.log(result)
             })
-          })
+          }
+        },
+      )
+    }
+
+    signTypedData.onclick(() => {
```

Comment:
```suggestion
    signTypedData.onclick = () => {
```
Shouldn't this be assigned the function? I'm not sure what's going on here by calling `onclick`, since it hasn't been given a value yet.

Reply:
I think this is what @whymarrh meant. See the other `onclick` listeners for examples of this.

Reply:
Aha, got it. Fixed!